scripts.md

```diff
@@ -126,6 +126,20 @@ mkdir -p example/en_app/outputs/dialog
 python scripts/dialog_json2xml.py example/en_app/outputs/dialogNew.json -d example/en_app/outputs/dialog/ -v
 ```
 
+## Add additional json to the workspace dialog_nodes part
+The `workspace_addjson.py` script takes the workspace as the -w parameter, directory with the workspace as the -d parameter, JSON file to be added as the -j parameter, and the target variable as the -t parameter. The script finds all occurrences of the target variable in the workspace["dialog_nodes"] and adds the JSON file there.
```

Comment:
target variable => target key... it should be changed everywhere... related to comment in test file

Reply:
fixed